hack/ramen-must-gather

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+
+function usage {
+	echo "ramen-must-gather [appNamespace [hubClusterName [managedClusterNames...]]]"
+	echo "default appNamespace is current namespace"
+	echo "default hubClusterName is 'hub'"
+	echo "default managedClusterNames are all clusters in your kubeconfig"
+	echo
+	echo "Output: A .tar.gz file containing the gathered cluster data"
+}
+
+# Compute default command line parameters
+# Default namespace is the current namespace
+DefaultNamespace=$(kubectl config view --minify \
+	-ojsonpath='{.contexts[0].context.namespace}')
```

Comment:
What is the default namespace? the logs we want are in multiple namespaces (application namespace, ramen-system, rook-ceph, etc.)